server/src/hardware/decoder/decoderfunction.hpp

```diff
@@ -52,12 +55,22 @@ class DecoderFunction : public Object
     Property<DecoderFunctionFunction> function;
     Property<bool> value;
 
+    //! \brief Timeout in seconds to unlatch (turn off) active momentary functions
+    //! If set to zero, functions will not be automatically turned off
+    //! Active only for \ref DecoderFunctionType::Momentary and \ref DecoderFunctionType::Hold
+    Property<int> timeoutSeconds;
```

Comment:
Traintastic preference is to use (u)intX_t types, in this case it can be unsigned as a negative value doesn't make sense.

Reply:
Done. It's now `uint16_t` in preparation for milliseconds.